src/main/options.js

```diff
@@ -81,10 +81,9 @@ function inferParser(filepath, plugins) {
     pluginsLoaded: true
   }).languages.find(
     language =>
-      typeof language.since === "string" &&
```

Comment:
`language.since !== null`? We should omit unreleased languages here.

Reply:
Good catch.